scene/gui/text_edit.cpp

```diff
@@ -1217,7 +1217,7 @@ void TextEdit::_notification(int p_what) {
 							color.a *= readonly_alpha;
 						}
 
-						if (str[j] >= 32) {
+						if (str[j] > 32) {
```

Comment:
This will break the `underlined` logic, it will no longer underline whitespace.

Reply:
ye I don't know why I assumed that's not needed. I just left this block as is was before and moved my if below. 

On the side note maybe tabs should be underlined too? Putting them in string and underlining it looks weird since they are not underlined:
![image](https://user-images.githubusercontent.com/9964886/56963438-39f30500-6b59-11e9-9010-7cb3496d649e.png)
But I think is very uncommon usecase (probably)

Reply:
@iwek7 If it's not too complicated to implement, sure.

Reply:
It is not but I don't want to do this in this PR - it's not related to this issue